client/js/components/user/DpCreateItem.vue

```diff
@@ -141,6 +141,11 @@ export default {
     }
   },
 
+  emits: [
+    'get-items',
+    this.customComponent[this.entity].resetEvent
```

Comment:
why only the reset-event? shouldn't the update-event be emited too?

Reply:
I thing because updateEvent don't use as emit
in lin 254 `this.$root.$emit(this.customComponent[this.entity].resetEvent)`

Reply:
ah, it use as computed and @[dynamicEvent] in component tag, then make it now
thank you!

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2437/commits/0aad14eda94f08d51094283b88a207ae1389566c